en/micro-integrator/docs/references/connectors/servicenow-connector/servicenow-connector-example.md

```diff
@@ -0,0 +1,250 @@
+# ServiceNow Connector Example
+
+The ServiceNow connector allows you to access the ServiceNow REST API through WSO2 EI. ServiceNow is a software platform that supports IT service management and automates common business processes. This software as a service (SaaS) platform contains a number of modular applications that can vary by instance and user. 
+
+It focuses on service-orientation toward the tasks, activities, and processes.
+
+## What you'll build
+
+This example explains how to use ServiceNow Connector to create records in a table and retrieve its information. Assume your organization uses ServiceNow support and you need to create an incident. In order to do that we can use the tableAPI which is a Rest API. This can be easily done using WSO2 ServiceNow connector. We can use the following API which is designed using the ServiceNow connector to create record in the incident table as well as to read the created incident ticket. 
```

Comment:
a table >> incident table of `serviceNow`?

Reply:
Whenever integration needs to raise an incident at `serviceNow`, above EI API can be called with required information.

Reply:
Done